inputs/text.vue

```diff
@@ -45,9 +46,10 @@
     :error="errorMessage"
     :disabled="isDisabled"
     iconPosition="right"
+    v-dynamic-events="customEvents"
     @input="update"
     @keydown-enter="closeFormOnEnter">
-    <attached-button slot="icon" :name="name" :data="data" :schema="schema" :args="args" @disable="disableInput" @enable="enableInput"></attached-button>
+    <attached-button slot="icon" :name="name" :data="data" :schema="schema" :args="args" @disable="disableInput" @enable="enableInput" v-dynamic-events="customEvents"></attached-button>
```

Comment:
Can you please break this element in multiple lines?

Reply:
👍